samples/zigbee/shell/README.rst

```diff
@@ -0,0 +1,166 @@
+.. _zigbee_shell_sample:
+
+Zigbee: Shell
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This :ref:`Zigbee <ug_zigbee>` shell sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the shell interface inside Zephyr's :ref:`zephyr:shell_api` for interaction.
+
+The sample can be used for several purposes, including:
+
+* Initial configuration of the network - forming a network as a coordinator, adding devices to the network with the install codes, setting the extended PAN ID.
+* Benchmarking - measuring time needed for message to travel from one node to another.
+
+Available CLI interfaces
+************************
+
+The CLI can be accessed by using the following interfaces:
+
+* UART
+* RTT
+
```

Comment:
Should we link to this page here? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_testing.html

This is what we have on the shell library page:
```
You can run the Zigbee shell commands after connecting and configuring the backend for testing.
For more information, see :ref:`gs_testing`.
```

Reply:
Added :slightly_smiling_face: